### ISSUE TITLE
Li TSS track data PMID: 25747261

### ISSUE THREAD

**Issue by Antonialock** [pombase/website#736]
It looks like there was some problem with this data. Can we bang the data up and see what needs changing @kimrutherford ?
ftp://ftp.pombase.org/external_datasets/HuaLiTSS/

These are the outstanding tasks from PB-2511:

> Upon closer inspection of these datasets, ALL represent TSS data (A TSS is only part of a core promoter). These cannot possibly represent promoters because a promoter would require a 'start' and an 'end' and we only have a single coordinate.
> 
> The only difference between the data in Supp 1 & 2 appears to be that supp2 refers to TSS for CDS which do not already have an annotated TSS.
> 
> 1.
> In the absence of start and end coordinates to represent promoters we should represent all of these as TSS, AND I see no logical reason why all of the data cannot be combined into a single track (it doesn't matter if its RNA or CDS for display purposes).
> 
> 2.
> For display, use only the geneID in the label (omit the _DP, MP etc from the label. At least some of the RNAs have _0. This should also be removed.
> 
> This subscript is confusing because:
> i) its explanation is hidden in a key which is only available via the pull down menu at the beginning of the track. It is so hidden that nobody will find it.
> ii) it requires an additional key which departs from the regular Ensembl feature keys colours (for protein RNA etc), but it does not represent tru biologically defined classes of promoter.
> iii) the information is somewhat arbitrary and confusing to overload the label with.
> Instead, include it as a description in the individual feature drop down.
> 
> i.e
> TSS distribution: Single dominant peak
> TSS distribution: Broad with a single dominant peak
> TSS distribution: Broad or bi multi peaks
> TSS distribution: Generally broad
> 
> No need for any colours or abbreviations
> NOTE THAT WITHOUT THE TEXT "TSS distribution"the existing descriptions are meaningless, because you have no way of knowing what these strings refer to.
> 
> 3.
> Define a single feature colour for "TSS", and if possible when the TSS track(s) is switched on and TSS are visible this should appear in the main "Gene legend'
> 
> 4.
> Remove the existing "Drop Down" key to the labels
> 
> 5. Some of the existing features have a score, but there is no score associated with this data. Which column did you get this from? I need to work out what it is before we decide if it is helpful to include it.
> 
> This should resolve all of the issues in PB-2509 and more.....
> 
> We will no longer have any data in the 'promoter' track.

#735

**Comment by mah11** [comment]
In #368 questions arose about the tracks in this directory:

LiTSS_TS1.bed
LiTSS_TS2.bed
LiTSS_TS6.bed
new/LiTSS.bed
new/LiTSS_TS6.bed


I did some ticket necromancy, and my best resulting hypothesis is that we ended up with two tracks, one for each file in the "new" subdirectory.

We merged TS1 and TS2 because the only difference was that the genes in TS1 had 5' UTRs annotated and those in TS2 didn't (at publication time; I imagine few if any have changed since). We could even have a single track in JBrowse, because the difference between new/LiTSS.bed and new/LiTSS_TS6.bed is that one is protein-coding genes and the other is ncRNA genes.

From a very quick glance it does look like new/LiTSS.bed is equivalent to LiTSS_TS1.bed data plus LiTSS_TS6.bed data, and with some extensions on the systematic IDs removed. Similarly, new/LiTSS_TS6.bed looks like basically LiTSS_TS6.bed without ID extensions.

In case it helps with the data description stuff for configuration - Hua Li said 'the most used transcription start site for each gene is defined as "representative TSS" here'.

**Comment by ValWood** [comment]
thanks. and I learned a new word!

**Comment by kimrutherford** [comment]
I don't know what the status is of this dataset.  I've had a look at the BED files and they have the correct chromosome IDs, which makes things easier.

> Can we bang the data up and see what needs changing @kimrutherford ?

I can do that if it would be helpful.

**Comment by kimrutherford** [comment]
Is this the same as this issue?: #735

**Comment by Antonialock** [comment]
yes looks like it

**Comment by ValWood** [comment]
@Antonialock  is this a duplicate?

**Comment by ValWood** [comment]
closing, add anything required to #736

**Comment by mah11** [comment]
swapping open vs. closed because this has more information than #735

**Comment by ValWood** [comment]
PMID: 25747261

**Comment by ValWood** [comment]
we don't have anything hosted for this publication
https://www.pombase.org/reference/PMID:25747261
@mah11  you seem to have a handle on what is going on here  I will assign to you to implement or issue instructions

**Comment by ValWood** [comment]
Reading the ticket again it seems like 
"bang the data up" will get things moving...

**Comment by kimrutherford** [comment]
The files are ready to go.  Just needs some rows in the spreadsheet.

**Comment by kimrutherford** [comment]
> The files are ready to go.

I spoke too soon.  They need a little bit of processing and indexing but that won't take long. There are 6 files.  Will we be display all of them?:

LiTSS_TS1.bed
LiTSS_TS2.bed
LiTSS_TS6.bed
new/LiTSS.bed
new/LiTSS_TS6.bed

**Comment by ValWood** [comment]
I read this paper and I think we  need to display 3 files

Table S1  + Table S2 TSS protein coding
Table S6 TSS ncRNA
Table S7 TSS  not currently associated with a gene

but I don't know how this correlates with the provided files....
I would like to keep the RNA and protein coding separate for now, this will be helpful  for resolving the gene structures down the line.

**Comment by ValWood** [comment]
metadata for config file, still needs file names adding

Transcription   start sites | TSS protein coding |   | 972 h- | WT |   | h- | YES |   | vegetative growth |   | DeepCage | Li | 2015 | PMID:25747261 | E-MTAB-3188. | ArrayExpress |   | BED |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Transcription start sites | TSS ncRNA |   | TSS not associated with a gene | WT |   | h- | YES |   | vegetative growth |   | DeepCage | Li | 2015 | PMID:25747261 | E-MTAB-3188. | ArrayExpress |   | BED |  
Transcription start sites | TSS not associated with a gene |   | 972 h- | WT |   | h- | YES |   | vegetative growth |   | DeepCage | Li | 2015 | PMID:25747261 | E-MTAB-3188. | ArrayExpress |   | BED |

**Comment by mah11** [comment]
tl;dr: load these two:

- new/LiTSS.bed
- new/LiTSS_TS6.bed


> There are 6 files. Will we display all of them?
> 
> LiTSS_TS1.bed
> LiTSS_TS2.bed
> LiTSS_TS6.bed

I got myself convinced that we don't need to load these; their contents got further processed to generate the files in the /new/ subdirectory and we've kept them only because we're pack-rats.


> Table S1 + Table S2 TSS protein coding

This is new/LiTSS.bed

> Table S6 TSS ncRNA

This is new/LiTSS_TS6.bed

> Table S7 TSS not currently associated with a gene

We've never had a .bed file corresponding to From Table S7, and from the table contents and the accompanying text, I'm not convinced that there are TSSs identified in it. It appears to be a set of tag clusters (TCs) that hint at the existence of not-yet-annotated promoters for not-yet-annotated genes, but for which the start sites have not been determined.

**Comment by mah11** [comment]
Part of Val's metadata text file look borked. I was going to do the config entry myself because Val assigned this ticket to me. Should I carry on with it?

**Comment by ValWood** [comment]
> We've never had a .bed file corresponding to From Table S7, and from the table contents and the accompanying text, I'm not convinced that there are TSSs identified in it. It appears to be a set of tag clusters (TCs) that hint at the existence of not-yet-annotated promoters for not-yet-annotated genes, but for which the start sites have not been determined.

OK I'd assumed this was 'orphan TSS' that might be useful to identify missing genes, but if it isn't we can leave this out



This is the data I had.
I saved it out of gthe spreadsheet in the wrong format and not the original Excel is gone.

Data type	Track label	Assayed protein	Experimental background alleles	WT or mutant	Mutant alleles	Mating type	Conditions	Comment	Growth phase or response	Strand	Assay type	First author (surname)	Publication year	PubMed ID	Study ID	Database	Sample ID	Data file type	File name
data_type	label	assayed_gene_product	Background	Alleles	Mutant(s)	mating_type	conditions	comment	growth_phase_or_response	strand	assay_type	first_author	publication_year	pmed_id	study_id	DB	sample_id	data_file_type	short_description
Transcription start sites	TSS protein coding		972 h-	WT		h-	YES		vegetative growth		DeepCage	Li	2015	PMID:25747261	E-MTAB-3188.	ArrayExpress		BED	
Transcription start sites	TSS ncRNA		TSS not onassociated with a gene	WT		h-	YES		vegetative growth		DeepCage	Li	2015	PMID:25747261	E-MTAB-3188.	ArrayExpress		BED	
Transcription start sites	TSS not associated with a gene		972 h-	WT		h-	YES		vegetative growth		DeepCage	Li	2015	PMID:25747261	E-MTAB-3188.	ArrayExpress		BED

**Comment by ValWood** [comment]
You can continue with the config. Sorry I wasn't clear, I  wanted the file locations but the ticket moved on overnight and 
I only read the incoming e-mails.

I saved it out of the spreadsheet in the wrong format and not the original Excel is gone.
The data I had seems readable in the first file, but it has ^M characters in it.

**Comment by ValWood** [comment]
drat, this was the one I was meant to be doing https://github.com/pombase/website/issues/50

**Comment by mah11** [comment]
metadata done in https://github.com/pombase/pombase-config/commit/1003ecfe36e8a64f2f94a64d8ee632f226dae19a#diff-d1befa03c79ca0b84ecc488dea96bc68
